feature/policy_forwarding/encapsulation/otg_tests/inner_ttl_ipv4/README.md

```diff
@@ -387,7 +383,7 @@ paths:
   /network-instances/network-instance/protocols/protocol/config/identifier:
   /network-instances/network-instance/protocols/protocol/static-routes/static/prefix:
   /network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix:
-  /network-instances/network-instance/protocols/protocol/static-routes/static/next-hop-group/name:
+  # /network-instances/network-instance/protocols/protocol/static-routes/static/next-hop-group/name: # TODO: currently dont have support
```

Comment:
This path does exist, 

[/network-instances/network-instance/protocols/protocol/static-routes/static/next-hop-group/config/name](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-static-routes-static-next-hop-group-config-name)

you should be able to keep it here